docs/_data/repoStatus.yaml

```diff
@@ -0,0 +1,286 @@
+Accordion:
```

Comment:
@bennypowers hey i need some guidance here. We want to add a flag in this list to account for the current status for eg: New, Beta, Experimental.
I was thinking to make a change to this format and I came up with these the two options:
```
Alert:
  flag: New
  status:
    - Figma library: Ready
    - Responsive: Ready
    - RH Elements: Ready
    - WebRH: Ready
    - Documentation: Ready
 Accordion:
   flag: Beta
   status:
    - Figma library: Ready
    - Responsive: Ready
    - RH Elements: Ready
    - WebRH: Ready
    - Documentation: Ready
```
or

```
Accordion:
  flag: New
  progress:
    - name: Figma library
      status: Ready
    - name: Responsive
      status: Ready
    - name: RH Elements
      status: Deprecated
    - name: webRH
      status: In progress
    - name: Documentation
      status: Ready
```

let me know your thoughts on this

Reply:
we can do the same as we did with progress and make the top level an array

```yaml
- name: Accordion
  overallStatus: New
  libraries:
    - name: Figma
      status: Ready
```
etc

Reply:
keep in mind this list might need to support different types of entities
like token, element, and pattern

Reply:
@bennypowers thanks thats very helpful and makes sense.